src/Compilers/CSharp/Test/Emit/Emit/EndToEndTests.cs

```diff
@@ -27,16 +27,16 @@ public void OverflowOnFluentCall()
 
             switch (IntPtr.Size * 8) {
                 case 32 when isDebug:
-                    numberFluentCalls = 290;
+                    numberFluentCalls = 510;
```

Comment:
Don't think it needs investigation at this time.  Was mostly just curious if we knew off hand.  

It seems it could either be the JIT here or possibly our temporary optimizations.  We're less aggressive about collapsing certain types of temps in Debug mode hence could have bigger frames.  

The release number is more important here though.